site/en/setup.md

```diff
@@ -364,6 +384,10 @@ docker run -u $(id -u) -w /data -v $PWD:/data -- ocrd/all:maximum ocrd-tesserocr
 Note that the CLI is exactly the same, the only difference is the prefix to instruct Docker, as [explained above](#mini-medi-maxi)
 
 ## Installation of OCR-D Research Data Repository
+Optionally, you can also install the OCR-D research data repository. It collects all versions of documents and (intermediate) results created
```

Comment:
[the OCR-D research data repository](https://github.com/OCR-D/repository_metastore/)